Gibe.Navigation.Umbraco/Gibe.Navigation.Umbraco.csproj

```diff
@@ -1,323 +1,81 @@
-    <Compile Include="NodeTypes\UrlNodeType.cs" />
-    <Compile Include="UmbracoNavigationProvider.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="UmbracoNodeService.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="Gibe.Navigation.Umbraco.nuspec" />
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Gibe.Navigation\Gibe.Navigation.csproj">
-      <Project>{f585d1e9-74d1-49fc-8700-dae8c244bc25}</Project>
-      <Name>Gibe.Navigation</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets" Condition="Exists('..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets')" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets'))" />
-  </Target>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<PackageId>Gibe.Navigation.Umbraco</PackageId>
+		<PackageVersion>8.0.0</PackageVersion>
+		<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+		<PackageTags>Gibe Navigation Umbraco</PackageTags>
+		<Authors>Gibe Digital Ltd</Authors>
+		<Version>8.0.0</Version>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net472</TargetFramework>
+	</PropertyGroup>
+	<ItemGroup>
+	  <PackageReference Include="Gibe.UmbracoWrappers" Version="2.0.2" />
+	  <PackageReference Include="JetBrains.Annotations" Version="2021.3.0" />
+	  <PackageReference Include="Moq" Version="4.16.1" />
+	  <PackageReference Include="NUnit" Version="3.13.2" />
+	  <PackageReference Include="NUnit.ConsoleRunner" Version="3.6.1" />
+	  <PackageReference Include="UmbracoCms.Core" Version="8.1.0" />
+	  <PackageReference Include="UmbracoCms.Web" Version="8.1.0" />
+	</ItemGroup>
+	<ItemGroup>
+	  <ProjectReference Include="..\Gibe.Navigation\Gibe.Navigation.csproj" />
+	</ItemGroup>
+	<ItemGroup>
+	  <None Remove="C:\Users\steve.temple\.nuget\packages\umbraco.sqlserverce\4.0.0.1\build\..\runtimes\win-x86\native\**" />
```

Comment:
I guess it doesn't matter but should these be referencing your local machine like this?

Reply:
How bizarre?!